src/settings.rs

```diff
@@ -30,22 +30,28 @@ impl Settings {
             match key.as_str() {
                 "package_verifier_enable" => {
                     if value != "1" {
-                        warn!("Scanning apps with Google Play Protect has been disabled");
+                        warn!("Google Play Protect has been turned off programmatically");
```

Comment:
```suggestion
                        warn!("Google Play Protect is turned off");
```

Maybe it was turned off manually.